### ISSUE TITLE
Tutorial Breaks

### ISSUE THREAD

**Issue by Antonic111** [PrehistoricKingdom/feedback#621]
When in the tutorial I was making the mammoths and kind of rushed placing 2 of them before it told me to place the 5, now I'm stuck with 3/5 placed and cant make more or delete the ones I already placed, you should maybe add a check to see if this happens during the tutorial, also adding what keys are needed to press during tutorial would be nice, I was extremely confused when it was asking me to make a group edit did not know I had to double click the feeder.

**Comment by Luna-Amina** [comment]
I have also encountered this exact problem by being too hasty in placing a mammoth, now he is stuck and not clickable
![image](https://user-images.githubusercontent.com/104608043/165877592-066f7460-78ba-4726-92c9-52ba954d4dfd.png)

**Comment by TiggDarkfury** [comment]
Yeah, same her. Will have to restart and do the complete tutorial again.. :(

**Comment by janjj** [comment]
> When in the tutorial I was making the mammoths and kind of rushed placing 2 of them before it told me to place the 5, now I'm stuck with 3/5 placed and cant make more or delete the ones I already placed, you should maybe add a check to see if this happens during the tutorial, also adding what keys are needed to press during tutorial would be nice, I was extremely confused when it was asking me to make a group edit did not know I had to double click the feeder.

i had the same issues...
